starwars/src/App.js

```diff
@@ -33,7 +38,8 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
-      </div>
+       {this.state.starwarsChars.map(chars => <div className="newChars"><p> Name: {chars.name}</p> <p>Eye-color: {chars.eye_color}</p><p> Hair-color: {chars.hair_color}</p> <p>Birth Year: {chars.birth_year}</p></div>)}
```

Comment:
Well done mapping over your state data and returning values for that data! You got elements rendered on the screen 👍